packages/thumbprint-react/components/Calendar/index.tsx

```diff
@@ -170,39 +145,41 @@ export default function Calendar({
 
     const selectedDays = normaliseValue(value);
 
-    const modifiers: Partial<Modifiers> = {};
+    const modifiers: Record<string, Matcher> = {};
+    const modifiersClassNames: Record<Modifier, string> = {};
 
     if (typeof daysThemeDotIndicator === 'function') {
         modifiers['theme-dot'] = daysThemeDotIndicator;
+        modifiersClassNames['theme-dot'] = 'rdp-day--theme-dot';
     }
 
     if (typeof daysThemeStrikeout === 'function') {
         modifiers['theme-strikeout'] = daysThemeStrikeout;
+        modifiersClassNames['theme-strikeout'] = 'rdp-day--theme-strikeout';
     }
 
     return (
         <div className={styles.root}>
             <DayPicker
-                disabledDays={disabledDays || undefined}
+                disabled={disabledDays || undefined}
                 fromMonth={get(disabledDays, 'before', null)}
                 toMonth={lastMonth}
-                month={month || selectedDays[0]}
```

Comment:
Why was this prop removed? Seems like they still use it in v8: https://react-day-picker.js.org/basics/navigation

Reply:
Good point! I think I missed it from type check. Updated to `month `.👍